AGENTS.md

```
@@ -0,0 +1,691 @@
+# AGENTS
+
+- Quick notes for working on this repo (Go Redis Work Queue) 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)